BattleNetwork/bnEntity.h

```diff
@@ -132,6 +137,8 @@ class Entity :
   EventBus::Channel channel; /*!< Our event bus channel to emit events */
   MoveEvent currMoveEvent{};
   VirtualInputState inputState;
+  AppliedStatus previousStatus{ "None", frames(0) }; /*!< The previous status, Stun or Freeze, applied when time freeze started.*/
```

Comment:
change these 2 to use the bit flag e.g. `{ Hit::None, frames(0) };`

Reply:
The requested changes have been made.